factory.js

```diff
@@ -0,0 +1,45 @@
+const states = require('./states.js');
```

Comment:
So when demoing the other day I noticed something in this file. Using an argument like `['codes']` in these constructors will result in an error if we try to do anything in the module builder, since it does not expect to have an array of strings but rather an array of objects.

For now, I think what we should do is replace any arguments like `['codes']` or `['series']` or whatever with just an empty array. Eventually we may try to put something in this array, but let's just leave it blank for now